src/schemas/publishCatalogItem/schema.tsx

```diff
@@ -24,14 +24,16 @@ const publishCatalogItemSchema = {
       description:
         'Specific stipulations to consider and known behaviors to be aware of when using this design.',
       format: 'textarea',
-      'x-rjsf-grid-area': 12
+      'x-rjsf-grid-area': 12,
+      "x-encode-in-uri": true
     },
     pattern_info: {
       type: 'string',
       title: 'Description',
       description: 'Purpose of the design along with its intended and unintended uses.',
       format: 'textarea',
-      'x-rjsf-grid-area': 12
+      'x-rjsf-grid-area': 12,
+      "x-encode-in-uri": true
```

Comment:
@Yashsharma1911  these are native to rjsf right ? . if not where are we documenting these . we should be using a uiSchema and not putting the presentation specific properties in the data structure schema

Reply:
I agree @aabidsofi19 we can move it to uiSchema, we are tracking them here - https://github.com/layer5io/sistent/blob/master/src/schemas/readme.md#custom-properties

Reply:
Also in RJSF section in Meshmap developer guide

Reply:
Ops sorry, I mean "Kanvas"